exporter/signozclickhousemetrics/exporter.go

```diff
@@ -237,6 +273,9 @@ func (c *clickhouseMetricsExporter) Shutdown(ctx context.Context) error {
 	if c.cacheRunning {
 		c.cache.Stop()
 	}
+	if c.usageCollector != nil {
+		c.usageCollector.Stop()
+	}
```

Comment:
Write a test for this and ensure on shutdown, usage is reported is correctly and successfully.